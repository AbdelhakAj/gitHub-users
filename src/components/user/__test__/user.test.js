import React from "react";
import renderer from "react-test-renderer";
import User from "../user";

describe("<User />", () => {
  it("renders User", () => {
    const component = renderer.create(
      <User data={{ id: "", name: "", imgURL: "" }} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
