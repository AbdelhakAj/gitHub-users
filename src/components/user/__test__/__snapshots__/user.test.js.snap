// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User /> renders User 1`] = `
<div
  className="user"
>
  <div
    className="user-info"
  >
    <img
      alt="profile"
      className="user-img"
      src="https://www.bsn.eu/wp-content/uploads/2016/12/user-icon-image-placeholder-300-grey.jpg"
      width="50"
    />
    <div
      className="user-details"
    >
      <h2
        className="user-name"
      >
        
      </h2>
      <span
        className="user-id"
      >
        
      </span>
    </div>
  </div>
  <button
    className="user-repos-btn"
    onClick={[Function]}
  >
     
    view repos
     
  </button>
</div>
`;
